qa/L0_output_validation/lt_op_val_client.py

```diff
@@ -65,7 +65,7 @@ def test_name(self):
         msg = response.json()["error"]
         self.assertTrue(
             msg.startswith(
-                "Request for unknown model: 'libtorch_name_1_float32' has no available versions"
+                "unexpected inference output 'OUTPUT__0' for model 'libtorch_name_1_float32'"
```

Comment:
As @rmccorm4  suggested, add a comment below L60 on how the naming convention violation is being handled.

Reply:
@rmccorm4:

1. Done ✔️ 
2. I have removed the test since it looks like it is no longer required after naming convention updated.